tests/ztest_test.go

```diff
@@ -0,0 +1,30 @@
+package tests
```

Comment:
ztests are also being run during `make test-heavy`. https://circleci.com/gh/brimsec/zq/2970 . I understand we're in a changing world.